code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -354,6 +350,45 @@
 	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/syndi
 	jetpack = /obj/item/tank/jetpack/suit
 
+/obj/item/clothing/suit/space/hardsuit/syndi/RemoveHelmet()
+	. = ..()
+	//Update helmet to non combat mode
+	var/obj/item/clothing/head/helmet/space/hardsuit/syndi/syndieHelmet = helmet
+	syndieHelmet.activate_combat_mode()
+	syndieHelmet.update_icon()
+	for(var/X in syndieHelmet.actions)
+		var/datum/action/A = X
+		A.UpdateButtonIcon()
+	//Update icon
+	icon_state = "hardsuit[syndieHelmet.on]-[syndieHelmet.item_color]"
+	update_icon()
+	//Update the suit to non-combat mode
+	activate_combat_mode()
```

Comment:
```suggestion
	//Update the suit to non-combat mode
	activate_combat_mode()
	//Update icon
	icon_state = "hardsuit[syndieHelmet.on]-[syndieHelmet.item_color]"
	update_icon()
```

`syndieHelmet.on` is changed at `activate_combat_mode()`. This causes a minor visual bug where the hardsuit looks likes EVA mode while being in combat mode (the differences are barely recognizable anyway)